packages/next/src/index.test.ts

```diff
@@ -109,4 +114,21 @@ describe('Next', () => {
       expect(getIdTokenClaims).toHaveBeenCalled();
     });
   });
+
+  describe('handleSignOut', () => {
+    it('should redirect to Logto sign out url', async () => {
+      const client = new LogtoClient(configs);
+      await testApiHandler({
+        handler: client.handleSignOut(),
+        url: '/api/sign-out',
+        test: async ({ fetch }) => {
+          const response = await fetch({ method: 'GET', redirect: 'manual' });
```

Comment:
Why is `redirect: 'manual'` needed in here?

Reply:
Because the real sign out url is not available in test environment. We just need to check the url that is going to redirect is corret.

Reply:
ok